eurocropsml/dataset/dataset.py

```diff
@@ -103,12 +102,12 @@ def _format_dates(
                 for month, count, idx in zip(unique, unique_counts, unique_indices):
                     if count == 1:
                         month_data[month] = data[idx]
-                    else:
+                    elif preprocess_config.satellite == "S2":
                         try:
                             cloud_probs = np.apply_along_axis(
                                 partial(
                                     find_clouds,
-                                    band4_idx=EUROCROPS_S2BANDS.index("B4"),
+                                    band4_idx=S2_BANDS.index("04"),
```

Comment:
Make sure to also adjust the value of `remove_bands` inside the meta-learning repo once this is merged and published.